manifests/exception.pp

```diff
@@ -173,10 +182,23 @@
         $netsh_command = "C:\\Windows\\System32\\netsh.exe firewall ${fw_action} ${fw_command} name=\"${display_name}\" mode=${mode} ${allow_context}"
       }
       default: {
+        $mode = $enabled ? {
+          'yes' => 'yes',
+          'no'  => 'no',
+          true  => 'yes',
```

Comment:
I'm pretty sure there is a stdlib function for yes/true translation

Reply:
@juniorsysadmin There is str2bool that will convert (all kinds of) strings to boolean, however, what we require here is a mapping to 'yes'/'no', since that is what we require on the command line then. So, the variable could be defined as String and later converted with str2bool, but in the end we'd still have to do the mapping from bool to string...